api/desecapi/pdns_change_tracker.py

```diff
@@ -82,7 +82,7 @@ def axfr_required(self):
             return True
 
         def pdns_do(self):
-            salt = '%016x' % random.randrange(16 ** 16)
+            salt = '%016x' % secrets.randbelow(16 ** 16)
```

Comment:
should be equivalent to the more clear `secrets.token_hex(8)  # 16 hex chars`

Reply:
done